packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginCardWithValidation.js

```diff
@@ -22,7 +22,7 @@ class LoginCardWithValidation extends React.Component {
     form: {
       showError: this.props.showError,
       submitError: this.props.submitError,
-      disableSubmit: this.props.disableSubmit,
+      disableSubmit: this.props.validate,
```

Comment:
Can you set this to `true`? as it would only be accessible if the component is validated

Reply:
yes, thanks